jxl/src/entropy_coding/hybrid_uint.rs

```diff
@@ -53,11 +53,11 @@ impl HybridUint {
             return Ok(symbol);
         }
         let bits_in_token = self.lsb_in_token + self.msb_in_token;
-        let nbits =
+        let mut nbits =
             self.split_exponent - bits_in_token + ((symbol - self.split_token) >> bits_in_token);
-        if nbits > 29 {
-            return Err(Error::IntegerTooLarge(nbits));
-        }
+        // To match the behaviour of libjxl, instead of returning an error here, we limit the
```

Comment:
What if we limit the size to 31 instead?

Reply:
That works for lossless_pfm image. Done.